sites.json

```diff
@@ -862,6 +862,16 @@
     },
 
     {
+        "name": "cda",
+        "url": "https://www.cda.pl/kontakt",
+        "difficulty": "hard",
+        "notes": "You need to contact with support",
```

Comment:
Does this need to be done via email?

Reply:
Unfortunately yes
There is an email at the bottom of https://www.cda.pl/kontakt